frontend/src/components/eMIB/EditActionDialog.jsx

```diff
@@ -121,7 +121,11 @@ class EditActionDialog extends Component {
             <Modal.Body style={styles.modalBody}>
               {actionType === ACTION_TYPE.email && <EditEmail onChange={this.editAction} />}
               {actionType === ACTION_TYPE.task && (
-                <EditTask emailId={this.props.emailId + 1} emailSubject={this.props.emailSubject} />
+                <EditTask
+                  emailId={this.props.emailId + 1}
```

Comment:
As discussed offline, we need the '+1' thing because it is used to display the email number in the add task form and the ID starts with 0, not 1.

According to that, I renamed the prop variable and added the '+1' thing inside the _EditTask.jsx_ component itself and removed it from _EditActionDialog.jsx_ component to avoid confusion.

See commit #https://github.com/code-for-canada/project-thundercat/pull/193/commits/9d3ae3cb6aaba48bb94b7ff58700f799530bf267